### DISCUSSION TITLE
cargo-mutants 1.0.0

### DISCUSSION THREAD

**Discussion by sourcefrog** [sourcefrog/cargo-mutants#83] [Announcements]
A 1.0 release to celebrate that with the addition of workspace handling, cargo-mutants gives useful results on many Rust projects.

- New: Supports workspaces containing multiple packages. Mutants are generated for all relevant targets in all packages, and mutants are subject to the tests of their own package.  `cargo mutants --list-files --json` and `cargo mutants --list --json` now includes package names for each file or mutant.

- Improved: Generate mutations in `cdylib`, `rlib`, and ever other `*lib` target. For example, this correctly exercises Wasm projects.

- Improved: Write `mutants.out/outcomes.json` after the source-tree build and baseline tests so that it can be observed earlier on.

- Improved: `mutants.out/outcomes.json` includes the commands run.

<hr /><em>This discussion was created from the release <a href='https://github.com/sourcefrog/cargo-mutants/releases/tag/cargo-mutants-1.0.0'>cargo-mutants 1.0.0</a>.</em>

**Comment by dmoka** [comment]
Huge congrats on the 1.0.0 release, and many thanks for your hard work!

This library is getting more and more awesome! :muscle:
